bundles/org.eclipse.swt/Eclipse SWT WebKit/gtk/org/eclipse/swt/browser/WebKit.java

```diff
@@ -795,7 +797,7 @@ public void create (Composite parent, int style) {
 				break;
 			}
 			case SWT.KeyDown: {
-				if (!disableBrowserSearch && event.keyCode == 'f' && (event.stateMask & SWT.CTRL) == SWT.CTRL) {
+				if (!disableBrowserSearch && enableSearch && event.keyCode == 'f' && (event.stateMask & SWT.CTRL) == SWT.CTRL) {
```

Comment:
disable Browser search should not be required here if clients want to opt in they should provide some kind of own way to pass `SWT.SEARCH` or not to the browser instance created.

Reply:
Makes me wonder if we should use the VM option to just enable search in every `Browser` (aside from hover pop-ups, i.e. in a `SWT.TOOL` `Shell`... WDYT @iloveeclipse , @laeubi ?

Reply:
I think having the JVM option can be removed. The search will be disabled by default and if one wants it it can be enabled for that particular widget. This would also be backward compatible.

Reply:
That is how we would use the patch in its current form, yes.